test/withGL.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+function withGL(init, fn)
+{
+    if (!PIXI.utils.isWebGLSupported())
+    {
+        return undefined;
+    }
+
+    const renderer = init();
+
+    try
+    {
+        return fn(renderer);
+    }
+    finally
+    {
+        PIXI.utils.destroyTextureCache();
+        renderer.destroy();
```

Comment:
One more example: we can change some renderer options on fly and run test multiple times instead of one, like what I do in "container.addChild" tests.